packages/common/src/chains/latticeTestnet.ts

```diff
@@ -3,7 +3,6 @@ import type { MUDChain } from "./types";
 export const latticeTestnet = {
   name: "Lattice Testnet",
   id: 4242,
-  network: "lattice-testnet",
```

Comment:
for anyone reading: the `network` key got removed from `Chain` in latest viem (I think this was a ethers carry-over)

Reply:
The network field was removed in [this commit](https://github.com/wevm/viem/commit/dd8f0c77011d6b75d4440f033366c9edd3214f77#diff-e342de8b8f37c8c14edefc2e0e74e7a34b156a5022e58564cf7da481db8efeaa). It suggests using the id field instead when necessary.